Sources/PackageModel/SwiftSDKs/SwiftSDK.swift

```diff
@@ -726,7 +734,10 @@ public struct SwiftSDK: Equatable {
             // Append the host toolchain's toolset paths at the end for the case the target Swift SDK
             // doesn't have some of the tools (e.g. swift-frontend might be shared between the host and
             // target Swift SDKs).
-            hostSwiftSDK.toolset.rootPaths.forEach { swiftSDK.append(toolsetRootPath: $0) }
+            let rootPaths = Set(swiftSDK.toolset.rootPaths)
```

Comment:
Avoids duplicates in newly formed `swiftSDK.toolset.rootPaths`. Not strictly related to the original issue, but makes it easier to write a test verifying how a target Swift SDK is derived with `--triple` CLI option, since no special casing is needed to account for this duplication.